package.json

```diff
@@ -26,9 +26,10 @@
     "ember-cli-inject-live-reload": "^1.3.0",
     "ember-cli-qunit": "0.3.13",
     "ember-cli-uglify": "^1.0.1",
+    "ember-disable-prototype-extensions": "^1.0.0",
     "ember-disable-proxy-controllers": "^1.0.0",
     "ember-export-application-global": "^1.0.2",
-    "ember-disable-prototype-extensions": "^1.0.0",
+    "ember-legacy-views": "0.2.0",
```

Comment:
Sorry, I didn't see the re-ordering, and it is actually better to keep it asc ordered, so my bad.
For the legacy view, if it is just for the `select`, then again my bad since the select is used in the demo only, so nevermind. Anyway, I am already working locally on doing all changes for Ember `1.13`, including changing the `{{each item in array}}` into `{{each array as |item|}}`. Thanks a lot of the help, I am going to push as soon as I got done with that so you can continue in that direction if it's still broken. Sorry I thought I was clear when I said that I was working on updating the `bower.json` and the related stuffs